pkg/apis/uiplugin/v1alpha1/types.go

```diff
@@ -31,20 +31,68 @@ type UIPluginList struct {
 	Items           []UIPlugin `json:"items"`
 }
 
-// +kubebuilder:validation:Enum=Dashboards
+// +kubebuilder:validation:Enum=Dashboards;Logging
 type UIPluginType string
 
 const (
-	// Dashboards deploys the Dashboards Dynamic Plugin for OpenShift Console.
+	// TypeDashboards deploys the Dashboards Dynamic Plugin for OpenShift Console.
 	TypeDashboards UIPluginType = "Dashboards"
+
+	// TypeLogging deploys the Logging View Plugin for OpenShift Console.
+	TypeLogging UIPluginType = "Logging"
 )
 
+// LoggingConfig contains options for configuring the logging console plugin.
+type LoggingConfig struct {
+	// LokiStack points to the LokiStack instance of which logs should be displayed.
+	// It always references a LokiStack in the "openshift-logging" namespace.
```

Comment:
AFAIU we can only have 1 logging UIPlugin object in the cluster but we could have multiple LokiStack objects. Is this correct?

Reply:
FYI we don't support multiple LokiStack objects through this UIPlugin in OCP. Although NetObserv saves partially data in their own LokiStack instance, they still use their own UI based on a custom console plugin